tests/chocolatey-tests/choco-template.Tests.ps1

```diff
@@ -35,6 +35,10 @@ Describe "choco <_>" -ForEach @(
             $Output.Lines | Should -Contain 'msi 1.0.2'
             $Output.Lines | Should -Contain 'zip 1.0.0'
         }
+
+        It "Displays how many custom templates are available" {
+            $Output.Lines | Should -Contain "2 Custom templates found at $env:ChocolateyInstall\templates"
+        }
     }
 
     Context "Running with list subcommand" {

```

Comment:
This wasn't from your change, but seeing your change I'm struck by the duplication of these two contexts (running with list and with no subcommand). We (and by that I mean me...) should perhaps create an issue to combine these into a single context.

Reply:
True, should probably look at a way to reduce that duplication. I don't think that should be something we should take into account right now though.

Reply:
I've opened a new issue to track this. Will update the issue with proper details later.